docs/UserGuide.md

```diff
@@ -758,11 +797,63 @@ Exit the application.
 
 ### Export Data
 
-<!-- TODO: fill this in -->
+> `export {file_path} [/overwrite]`
+
+Export all modules progress to a JSON-format file. 
+
+- <span style="color:#e46c0a">`file_path`</span> : The path of the file to export the modules progress to
+  - Must be relative to Le Tracker's default saving directory (:exclamation:The default saving directory is `[JAR 
```

Comment:
As per UG command syntax rules, probably best to use `{JAR_file_location}/data` instead of square brackets which represent optional items.

This issue is repeated multiple times in your edit.